Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -50,8 +51,15 @@ public override SceneQueryType SceneQueryType
         /// <remarks>
         /// Only [NearInteractionGrabbables](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionGrabbable) in one of the LayerMasks will raise events.
         /// </remarks>
+        [Obsolete("Use PrioritizedLayerMasksOverride instead")]
         public LayerMask[] GrabLayerMasks => grabLayerMasks;
 
+        public override LayerMask[] PrioritizedLayerMasksOverride
```

Comment:
So yeah, following up on what I was thinking below, I think we can go this route. We could also go down the route of just re-setting `PrioritizedLayerMasksOverride` to be equal to `GrabLayerMasks` any time `PrioritizedLayerMasksOverride` is null when we're running through `OnPreSceneQuery`. That'd allow devs to still override this while falling back to `GrabLayerMasks` instead of overwriting it all the way through. I _think_ I'm good either way, but wanted to put it out there to see what people thought.

Reply:
Good point Kurtis. I have a preference for keeping the existing behavior.

The current external facing behavior is that the pointer will use the grab layer mask, unless somebody sets PrioritizedLayerMasksOverride. Then when PrioritizedLayerMasksOverride goes to null, it defaults back to GrabLayerMasks. Let's think about this from POV of cost/benefit to the end-user of MRTK. What's the benefit of changing the API? The cost currently is we are changing behavior, so people may lose time reading release notes / debugging if code that they used to have now changes.

It seems there is not much benefit to changing the behavior to the end-user (though for MRTK devs there is benefit of simplifying the code, but we are just a few people compared to thousands that use MRTK). The cost is people may hit a tricky bug if we change the behavior.